File: app/src/main/java/com/ababilx/routinescrapper/domain/model/ClassSlot.kt
package com.ababilx.routinescrapper.domain.model

data class ClassSlot(
    val day: RoutineDay,
    val slot: Int,
    val start: String,
    val end: String,
    val course: String,
    val group: String,
    val teacher: String,
    val room: String,
)

File: lib/domain/model/class_slot.dart
import 'routine_day.dart';

class ClassSlot {
  const ClassSlot({
    required this.day,
    required this.slot,
    required this.start,
    required this.end,
    required this.course,
    this.courseTitle = '',
    required this.group,
    required this.teacher,
    required this.room,
  });

  final RoutineDay day;
  final int slot;
  final String start;
  final String end;
  final String course;
  final String courseTitle;
  final String group;
  final String teacher;
  final String room;
}
